packages/core/integration-tests/test/image.js

```diff
@@ -0,0 +1,28 @@
+import assert from 'assert';
+import {bundle, distDir, outputFS} from '@parcel/test-utils';
+import path from 'path';
+
+describe('image', function() {
+  it('Should be able to resize images', async () => {
+    await bundle(path.join(__dirname, '/integration/image/resized.js'));
+
+    let dirContent = await outputFS.readdir(distDir);
+    console.log(dirContent);
+
+    assert.equal(dirContent.length, 8);
```

Comment:
This should output an image for each format, it just outputs one.
I tried updating the key but it does not seem to work. Not entirely sure how I should fix this. I thought it might be a bundling thing but the assets should be isolated?